profiles/SampleMoonMage-setup.yaml

```diff
@@ -0,0 +1,129 @@
+---
+# https://elanthipedia.play.net/mediawiki/index.php/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Sorcery (Does not include Lunar Magic)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# You will need a runestone in your inventory to train sorcery and arcana. Grab one from any donation shelf.
+# You will need to add more magic and survival skills depending on your circle
+# However, you will probably end up training them all since they each have very useful spells
+# The other Magic skills are: Targeted Magic, Warding, Debilitation, Utility
+
+# Hunting settings
+training_manager_hunting_priority: true
```

Comment:
Not really, weapons will lock quick and take awhile to buffer down, I might make my stop ons and priority skills shield usage and one of my weapon skills. percept is kind of hit or miss for locking in combat, as is attunement.